### PR TITLE
Update README.md

### LINKED ISSUE #23: Complete the README.md, Contribution_Guidelines.md for new participants.
I will be adding a link to google docs after discussion with mentors and Project Admin, You have to simply refer to that doc file and add 
* ~~Details about the project, i.e., AIM and Implementation in README.md. The style of README is left for your discretion.~~ Assigned to @Subhangini 
* ~~Contributing_Guidelines.md needs to be created and add our suggestions and info from the remaining portion of the doc file there.~~ -> Assigned to @AshuKV.

I need two different PR for this issue, and I will add a level1 tag over there or level2 in case I find the work actually decent and automated.

* A potential PR can be Directing any participant towards Countributing_Guidelines.md before he creates an issue.

Few <b> important</b> guidelines for you:
* Create hyperlinks like [this](https://github.com/CrewMates-Open-Source/EduHelp) and not like this one (https://github.com/CrewMates-Open-Source/EduHelp).
*  Use proper tags where ever necessary and your creativity is appreciated but don't overdo things unless you are looking for being a web designer.
* Please preview the .md file before making a PR.

### PR DESCRIPTION
**Pull request by Subhangini**
Created a informative an attractive readme file 
Fixes #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,94 @@
-# EduHelp
+<h1 align="center"> EduHelp</h1>
```
Comment (shankhanil007) [inline README.md:1]: The EduHelp gif is simply amazing.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,94 @@
...
+**EduHelp** simply mean to make it easier for you to *record any online session*, be it an online lecture, meetings, youtube tutorials, etc. But we are not simply making a web extension or desktop app that records your screen or gives you an audio file in the end. We mean to *save the slide show* or screen presentation going on during those meets as well as the *text file of the entire conversation* that took place. This Project will not only help normal people but also *disabled persons* because the audio is converted to text and a file containing its summary is saved in .txt file as well as *braille scripted file*.The braille file can be given as an input to refreshable braille display (rbd) machines. The audio files can also be used by a blind person for taking note of what happened during class. The screenshots are stored in a .docx file and can be used by a deaf person for revisiting concepts they might miss during classes.
```
Comment (shankhanil007) [inline README.md:18]: Change "disabled persons" to "differently-abled people".

File: README.md [hunk 3]
```diff
@@ -1,12 +1,94 @@
...
 <h2 align="center"> AMAZING!!😮:star_struck:RIGHT?</h2>
 
+## Demo
+Want to know more about EduHelp. Just click the Image Below :point_down:
+<p align="center">
+  <a href="https://youtu.be/qgMU8F1QX5g"><img src="https://img.youtube.com/vi/qgMU8F1QX5g/maxresdefault.jpg" width=500 height=200></img></a>
```
Comment (shankhanil007) [inline README.md:24]: The video thumbnail is resized improperly.  The width is way more than the height. See our main branch video thumbnail and make changes accordingly.

**Comment by GrayFlash** [comment]
@Subhangini I am merging your PR, please make the recommended changes in a subsequent PR. We won't assign this to anyone else for the time being.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,94 @@
...
+  [![Issues closed](https://img.shields.io/github/issues-closed/CrewMates-Open-Source/EduHelp)](https://github.com/CrewMates-Open-Source/EduHelp/issues)
+  [![PRs open](https://img.shields.io/github/issues-pr/CrewMates-Open-Source/EduHelp)](https://github.com/CrewMates-Open-Source/EduHelp/pulls)
+  [![PRs closed](https://img.shields.io/github/issues-pr-closed/CrewMates-Open-Source/EduHelp)](https://github.com/CrewMates-Open-Source/EduHelp/pulls)
+  ![Repo size](https://img.shields.io/github/repo-size/CrewMates-Open-Source/EduHelp)
+  [![Open Source Love svg2](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
+ 
+[![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/built-with-love.svg)](https://forthebadge.com)
+
 ## What It Does ? :thinking:
```
Comment (shankhanil007) [inline README.md:17]: Don't capitalize the 1st letter of every word. Do it for only the 1st word. Rest all words should be in lower case.

File: README.md [hunk 5]
```diff
@@ -1,12 +1,94 @@
...
 <h2 align="center"> AMAZING!!😮:star_struck:RIGHT?</h2>
 
+## Demo
+Want to know more about EduHelp. Just click the Image Below :point_down:
+<p align="center">
+  <a href="https://youtu.be/qgMU8F1QX5g"><img src="https://img.youtube.com/vi/qgMU8F1QX5g/maxresdefault.jpg" width=500 height=200></img></a>
+</p>
+
+## What Do We Think Of Implementation ?
```
Comment (shankhanil007) [inline README.md:27]: Don't capitalize the 1st letter of every word. Do it for only the 1st word. Rest all words should be in lower case.

File: README.md [hunk 6]
```diff
@@ -1,12 +1,94 @@
...
+## Tech-Stacks 💻
+- For Website: `HTML`, `CSS`
+- For core functionalities: `Python`
+- For summarizer part: `NLP`
+- For different Speech audio being clustered: `ML`
+- `Electron.js` or `Tkinter` or any other Desktop App development stack.
+- For making all the functionalities available as an API: `Flask Backend`
+
+## How to get started?
+<h3 align="center">A beginner and don't know How and Where to start???</h3>
```
Comment (shankhanil007) [inline README.md:40]: Don't capitalize the 1st letter of every word. Do it for only the 1st word. Rest all words should be in lower case.

File: README.md [hunk 7]
```diff
@@ -1,12 +1,94 @@
...
+</p>
+Don't worry, You can refer to the following articles and also contact the Project Maintainers, in case you are stuck:
+
+- [Watch this video to get started, if you have no clue about open source](https://www.youtube.com/watch?v=c6b6B9oN4Vg)
+- [Forking a Repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+- [Cloning a Repo](https://help.github.com/en/desktop/contributing-to-projects/creating-a-pull-request)
+- [How to create a Pull Request](https://opensource.com/article/19/7/create-pull-request-github)
+- [Getting started with Git and GitHub](https://towardsdatascience.com/getting-started-with-git-and-github-6fcd0f2d4ac6)
+
+## How to Contribute?
```
Comment (shankhanil007) [inline README.md:52]: Don't capitalize the 1st letter of every word. Do it for only the 1st word. Rest all words should be in lower case.

File: README.md (hunk 1 above)
Comment (Subhangini) [reply README.md:1]: Thank you

File: README.md (hunk 2 above)
Comment (Subhangini) [reply README.md:18]: ok

**Pull request merged**